HISTORY.rst

```diff
@@ -2,6 +2,72 @@
 History
 =======
 
+3.0.0 (2023-10-27)
+------------------
+
+This new major release of ``unyt`` fixes a number of issues and adds new units.
+Major contributions include:
+
+* A number of new metric and non-metric units have been added
+  (`PR #441 <https://github.com/yt-project/unyt/pull/442>`_). Thank you
+  to John ZuHone (@jzuhone on GitHub) for the contribution.
+
+* The :ref:```@returns`` decorator<Checking Units>` now allows dimension-checking
+  of multiple return values (`PR #435 <https://github.com/yt-project/unyt/pull/435>`_).
+  Thank you to Daniel Bates (@db434 on GitHub) for the contribution.
+
+* A fix for for the LaTeX representation of Planck units
+  (`PR #379 <https://github.com/yt-project/unyt/pull/379>`_). Thank you to
+  Peter Hayman (@haymanpf on GitHub) for the contribution.
+
+* A fix for a bug that prevented the conversion of dimensionless arrays
+  to their corresponding `AstroPy Quantities <https://docs.astropy.org/en/stable/units/>`_
+  (`PR #437 <https://github.com/yt-project/unyt/pull/437>`_). Thank you to
+  Clément Robert (@neutrinoceros on GitHub) for the contribution.
+
+* A memory-management optimization obtained by avoiding unnecessary copies
+  in ``numpy.histogram*`` functions (`PR #425 <https://github.com/yt-project/unyt/pull/425>`_).
+  Thank you to Clément Robert (@neutrinoceros on GitHub) for the contribution.
```

Comment:
This is an optimisation for NEP-18 code, so it's not a change wrt v2.9.5, I would move this to the NEP-18 section